vumi_http_retry/service.py

```diff
@@ -0,0 +1,23 @@
+from twisted.web import server
+from twisted.python import usage
+from twisted.application import service, strports
+
+from vumi_http_retry.app import VumiHttpRetryServer
+
+
+class Options(usage.Options):
+    optParameters = [[
+        'port', 'p', '8080',
+        'Port to listen on'
+    ]]
+
+
+def makeService(options):
+    server = VumiHttpRetryServer()
+
+    site = server.Site(server.app.resource())
+    srv_service = service.MultiService()
```

Comment:
Err, more context: I went with creating a twistd plugin for the api instead of using the web plugin that I think klein exposes in case we want to add custom config options later (which we will probably need once we want a way of specifying a path to a config field).

I'm guessing there are things other than `MultiService` that can still allow us to accomplish that though, happy to go with what makes more sense here.